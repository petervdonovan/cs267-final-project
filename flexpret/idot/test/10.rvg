[mu []
[[file {Reset_Handler} {
  Reset_Handler:
    [id init]
    [single-core t5
      [pariterate-hart t1 of [cyclify a1 a2 [.. a0 3 to 19] times 3]
        {
          [critical-hart t0 t5
            {
              [sim-print t1 t4]
            }]
        }]]
    [id die]}]]
]
