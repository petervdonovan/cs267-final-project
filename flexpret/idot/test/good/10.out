Core-0:          3
Core-0:          5
Core-0:          7
Core-0:          9
Core-0:         11
Core-0:         13
Core-0:         15
Core-0:         17
Core-0:          4
Core-0:          6
Core-0:          8
Core-0:         10
Core-0:         12
Core-0:         14
Core-0:         16
Core-0:         18
Core-0:          3
Core-0:          5
Core-0:          7
Core-0:          9
Core-0:         11
Core-0:         13
Core-0:         15
Core-0:         17
Core-0:          4
Core-0:          6
Core-0:          8
Core-0:         10
Core-0:         12
Core-0:         14
Core-0:         16
Core-0:         18
Core-0:          3
Core-0:          5
Core-0:          7
Core-0:          9
Core-0:         11
Core-0:         13
Core-0:         15
Core-0:         17
Core-0:          4
Core-0:          6
Core-0:          8
Core-0:         10
Core-0:         12
Core-0:         14
Core-0:         16
Core-0:         18
Core-0 is done
Core-1 is done
Core-2 is done
Core-3 is done
Core-4 is done
Core-5 is done
Core-6 is done
Core-7 is done
Core-8 is done
Core-9 is done
Core-10 is done
Core-11 is done
Core-12 is done
Core-13 is done
Core-14 is done
Core-15 is done
All cores are done terminating
Assertion failed: Program terminated sucessfully
